### PR TITLE
move "openshift ex" -> "oc ex"

### PR DESCRIPTION
**Pull request by juanvallejo**
Moves the experimental command group to `oc` in order to break deps on `pkg/oc` in packages outside of that subtree. Part of https://github.com/openshift/origin/pull/17309 and https://github.com/openshift/origin/pull/17356

Release-Note: `Experimental commands moved from the "openshift" parent to the "oc" parent`

cc @deads2k @liggitt

### CODE REVIEW

**Comment by deads2k** [comment]
You need a release note on this.  

/lgtm

File: hack/lib/start.sh [hunk 1]
```diff
@@ -162,6 +162,7 @@ function os::start::internal::patch_master_config() {
 	local sudo=${USE_SUDO:+sudo}
 	cp "${SERVER_CONFIG_DIR}/master/master-config.yaml" "${SERVER_CONFIG_DIR}/master/master-config.orig.yaml"
 	oc ex config patch "${SERVER_CONFIG_DIR}/master/master-config.orig.yaml" --patch="{\"etcdConfig\": {\"address\": \"${API_HOST}:${ETCD_PORT}\"}}" | \
+	oc ex config patch - --patch="{\"pluginConfig\": {\"openshift.io/ImagePolicy\": {\"configuration\": {\"apiVersion\": \"v1\", \"executionRules\": [{\"matchImageAnnotations\": [{\"key\": \"images.openshift.io/deny-execution\", \"value\": \"true\"}], \"name\": \"execution-denied\", \"onResources\": [{\"resource\": \"pods\"}, {\"resource\": \"builds\"}], \"reject\": \"true\", \"skipOnResolutionFailure\": \"true\" }], \"kind\": \"ImagePolicyConfig\" }, \"location\": \"\"}}}" | \
```
Comment (juanvallejo) [inline hack/lib/start.sh]: @deads2k fixes tests in https://github.com/openshift/origin/pull/17473

Comment (deads2k) [reply hack/lib/start.sh]: > @deads2k fixes tests in #17473

You can save this commit for that pull.  You want to pick up that pull?

Comment (juanvallejo) [reply hack/lib/start.sh]: @deads2k 
> You can save this commit for that pull. You want to pick up that pull?

Sure, will start working on the pull now

**Pull request merged**